Jenkinsfile

```diff
@@ -30,14 +40,18 @@ podTemplate(
         }
 
         stage('Unit test') {
-            withEnv(["MIX_ENV=test"]) {
+            container('postgresql') {
+                sh("pg_isready -t 60 -h localhost -p 5432")
+            }
+
+            withEnv(["MIX_ENV=test", "DATABASE_URL=${DATABASE_URL}"]) {
                 sh("mix test")
             }
         }
 
         stage('Integration test') {
            withCredentials([string(credentialsId: 'elixir-omg_coveralls', variable: 'ELIXIR_OMG_COVERALLS')]){
-                withEnv(["MIX_ENV=test", "SHELL=/bin/bash"]) {
+                withEnv(["MIX_ENV=test", "SHELL=/bin/bash", "DATABASE_URL=${DATABASE_URL}"]) {
```

Comment:
I think DB url var isn’t necessary here. Consider revert.

Reply:
It is necessary.